docs/appendices/release-notes/6.0.0.rst

```diff
@@ -112,6 +112,10 @@ Scalar and Aggregation Functions
 Performance and Resilience Improvements
 ---------------------------------------
 
+- Improved the handling of temporary unavailable shards during read-only
```

Comment:
Seems that is also affects the logical-replication? should we mention this as well?

Reply:
I'm not sure if the error is propagated via logical-replication that way - one of the main areas is in the planner during routing resolving.

I mainly made the maybeTemporary change for consistency/completeness